DeviceInfo/DeviceInfoImplementation.cpp

```diff
@@ -0,0 +1,123 @@
+#include "DeviceInfoImplementation.h"
```

Comment:
As this is part of the interface definition, can we move the mapping to text also to the definition in the ThunderInterface/definitions section ?
That way all users of this interface will have the same textual mapping. Avoids confusion :-)